modules/task/backend.go

```diff
@@ -30,6 +30,12 @@ import (
 
 var gobEncoding = &GobEncoding{}
 
+const (
+	_initialized = iota
```

Comment:
@glibsm I don't see that in examples online. Let me know if you have examples. I personally don't like unknown, the name itself does not indicate what it is and is not meaningful here.